UDP co-ordinator fix

GBearUK opened the pull request:
The compare function used for ordering forwardingRequestList was wrong, which meant sometimes after the UDP co-ordinator inserted a forwarding request, if it tried to find it again with GetIndexFromKey, it would sometimes fail.

This meant that when it heard back from the proxy about a successful forwarding, it would sometimes not be able to find the original request, thus ignoring the response.

This resulted in clients which needed forwarding successfully getting it set up, but not hearing back about it.

GBearUK committed:
Fix for GetIndexFromKey failing on forwardingRequestList

The ordered list wasn't ordered!

Luke1410 commented:
Pull request integrated in https://github.com/SLikeSoft/SLikeNet (incorporated in SLikeNet 0.1.0).

Luke1410 commented:
Due to lack of a different way to reach out to you, I'm adding this one as a comment to your pull request:

Since we incorporated your pull request in SLikeNet, we'd like to offer you to be added to the acknowledgement section in the accompanying readme file.

By default the entry would look like:
GBearUK: https://github.com/GBearUK (pull request: RAKPR_67)

Please let me know (preferably by email) if we'd add such an entry in the next release of SLikeNet for you (and in case you'd prefer a different style, how it should look like). If there's no reply from your side, we'll **not** add such an entry to respect your privacy. Note that for those who decided not to be listed in the acknowledgement section and/or those we failed to contact, we added the following generic statement to cover these contributions:
"We'd also like to thank those contributors who have requested to remain
anonymous and/or those who we could not contact at all (due to lack of contact
information)."

GBearUK commented:
Sounds good thanks.

On 26 Sep 2017, at 23:40, Stefan Hett <notifications@github.com<mailto:notifications@github.com>> wrote:


Due to lack of a different way to reach out to you, I'm adding this one as a comment to your pull request:

Since we incorporated your pull request in SLikeNet, we'd like to offer you to be added to the acknowledgement section in the accompanying readme file.

By default the entry would look like:
GBearUK: https://github.com/GBearUK (pull request: RAKPR_67)

Please let me know (preferably by email) if we'd add such an entry in the next release of SLikeNet for you (and in case you'd prefer a different style, how it should look like). If there's no reply from your side, we'll not add such an entry to respect your privacy. Note that for those who decided not to be listed in the acknowledgement section and/or those we failed to contact, we added the following generic statement to cover these contributions:
"We'd also like to thank those contributors who have requested to remain
anonymous and/or those who we could not contact at all (due to lack of contact
information)."

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/facebookarchive/RakNet/pull/67#issuecomment-332356949>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AMXXhl6EcMe3H3Y6AsUwwixY4H4I2eg4ks5smX11gaJpZM4FIqt4>.

sharviltalati commented:
Hi Stefan,

I feel like I am accidentally on this thread and several other update emails on this theme?

Sharvil

From: Stefan Hett [mailto:notifications@github.com]
Sent: Tuesday, September 26, 2017 3:41 PM
To: facebookarchive/RakNet <RakNet@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Subject: Re: [facebookarchive/RakNet] UDP co-ordinator fix (#67)


Due to lack of a different way to reach out to you, I'm adding this one as a comment to your pull request:

Since we incorporated your pull request in SLikeNet, we'd like to offer you to be added to the acknowledgement section in the accompanying readme file.

By default the entry would look like:
GBearUK: https://github.com/GBearUK (pull request: RAKPR_67)

Please let me know (preferably by email) if we'd add such an entry in the next release of SLikeNet for you (and in case you'd prefer a different style, how it should look like). If there's no reply from your side, we'll not add such an entry to respect your privacy. Note that for those who decided not to be listed in the acknowledgement section and/or those we failed to contact, we added the following generic statement to cover these contributions:
"We'd also like to thank those contributors who have requested to remain
anonymous and/or those who we could not contact at all (due to lack of contact
information)."

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/facebookarchive/RakNet/pull/67#issuecomment-332356949>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AHqShdpj96KAblPMLFfKcnGyLc2MJXA4ks5smX1xgaJpZM4FIqt4>.

sharviltalati commented:
For ref:

[cid:image001.png@01D33760.55570230]


Sharvil

From: Sharvil Talati
Sent: Wednesday, September 27, 2017 7:14 AM
To: 'facebookarchive/RakNet' <reply@reply.github.com>
Subject: RE: [facebookarchive/RakNet] UDP co-ordinator fix (#67)

Hi Stefan,

I feel like I am accidentally on this thread and several other update emails on this theme?

Sharvil

From: Stefan Hett [mailto:notifications@github.com]
Sent: Tuesday, September 26, 2017 3:41 PM
To: facebookarchive/RakNet <RakNet@noreply.github.com<mailto:RakNet@noreply.github.com>>
Cc: Subscribed <subscribed@noreply.github.com<mailto:subscribed@noreply.github.com>>
Subject: Re: [facebookarchive/RakNet] UDP co-ordinator fix (#67)


Due to lack of a different way to reach out to you, I'm adding this one as a comment to your pull request:

Since we incorporated your pull request in SLikeNet, we'd like to offer you to be added to the acknowledgement section in the accompanying readme file.

By default the entry would look like:
GBearUK: https://github.com/GBearUK (pull request: RAKPR_67)

Please let me know (preferably by email) if we'd add such an entry in the next release of SLikeNet for you (and in case you'd prefer a different style, how it should look like). If there's no reply from your side, we'll not add such an entry to respect your privacy. Note that for those who decided not to be listed in the acknowledgement section and/or those we failed to contact, we added the following generic statement to cover these contributions:
"We'd also like to thank those contributors who have requested to remain
anonymous and/or those who we could not contact at all (due to lack of contact
information)."

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/facebookarchive/RakNet/pull/67#issuecomment-332356949>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AHqShdpj96KAblPMLFfKcnGyLc2MJXA4ks5smX1xgaJpZM4FIqt4>.

Luke1410 commented:
@sharviltalati You receive these notifications, because you are subscribed to these issues on GitHub. At the end of the mail, you'll see a link which directs you to the GitHub page. Please follow the instructions at the end of the mail to unsubscribe from these issues if you ware no longer interested in getting notifications for these.
